tests/integration/linux_bridge_test.py

```diff
@@ -210,6 +210,27 @@ def test_add_linux_bridge_with_empty_ipv6_static_address(port0_up):
     assertlib.assert_absent(bridge_name)
 
 
+def test_add_linux_bridge_with_empty_ipv6_static_address_with_stp(port0_up):
```

Comment:
What is special about `no IP addresses` in regards to STP? I would expect this to be an STP only related issue, no matter the IP settings.
Could you also remind me what it means?

Reply:
Tested on NM 1.12.6 and NM 1.20, the activation will wait if IPv6 enabled.